engine/api/src/main/scala/pl/touk/nussknacker/engine/api/process/ProcessObjectDependencies.scala

```diff
@@ -3,5 +3,4 @@ package pl.touk.nussknacker.engine.api.process
 import com.typesafe.config.Config
 import pl.touk.nussknacker.engine.api.namespaces.ObjectNaming
 
-case class ProcessObjectDependencies(config: Config,
-                                     objectNaming: ObjectNaming) extends Serializable
+case class ProcessObjectDependencies(config: Config, objectNaming: ObjectNaming) extends Serializable
```

Comment:
Why this reformatting? IMO before change it loos also ok

Reply:
Yeah yeah.. I wil move it to old place it was changed by automatic formatter.